files/dashboards/openshift-logging-dashboard.json

```diff
@@ -1795,6 +1795,133 @@
         "alignLevel": null
       }
     },
+    {
+      "collapse": false,
+      "height": "250px",
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "$datasource",
+          "fill": 1,
+          "id": 8,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "null as zero",
+          "percentage": false,
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "span": 12,
+          "stack": false,
+          "steppedLine": false,
+          "styles": [
+            {
+              "alias": "Time",
+              "dateFormat": "YYYY-MM-DD HH:mm:ss",
+              "pattern": "Time",
+              "type": "hidden"
+            },
+            {
+              "alias": "Total Bytes Logged in last 5 mins",
```

Comment:
5m seems short. 24 hours might be a more useful number, since the logging activity is likely to vary between night and day. Retention times are in days, weeks or months so a day seems like a reasonable unit to project from.

Reply:
@alanconway I think we can have dynamic value based on drop down. From 5 mins to 2 hours charts always shows values based on last 5 mins but beyond that it uses actual selected value which varies from 6 hours, 12 hours , 1 day,1 week, 2 weeks.

Reply:
I have added time-interval support so now user can select time-interval from drop down. Based on selected time-interval table values will be reflected.